policy/release/base_image_registries.rego

```diff
@@ -14,14 +14,15 @@ import future.keywords.in
 import data.lib
 
 # METADATA
-# title: Restrict registry of base images
+# title: Base image comes from permitted registry
 # description: >-
 #   The base images used when building a container image must come from a known set
-#   of trusted registries to reduce potential supply chain attacks. This policy
-#   defines trusted registries as registries that are fully maintained by Red Hat
-#   and only contain content produced by Red Hat.
+#   of trusted registries to reduce potential supply chain attacks. By default this
```

Comment:
I think generally yes, but I'll file another story rather than delay this further. Scope here is supposed to be titles and names. The bonus description tweaks are technically out of scope. (I did considered moving them to another PR, but let's keep it loose.)